### DISCUSSION TITLE
Indy Autonomous Challenge teams using plotjuggler

### DISCUSSION THREAD

**Discussion by lucasw** [facontidavide/PlotJuggler#552] [General]
There were brief glimpses of what the teams monitoring their vehicles were seeing, and there was a lot of plotjuggler on display:

![plotjuggler_and_custom_tool](https://user-images.githubusercontent.com/1334122/139539767-38d9b3b9-15bc-45bc-961d-7b0a921146c4.png)

![plotjuggler_imgui2](https://user-images.githubusercontent.com/1334122/139539829-a807ceaa-18cf-4b63-b5aa-81a46ca35c20.png)

There looks to be a custom rviz-like tool on the left of the screen that uses imgui (https://github.com/ocornut/imgui)

---

![plotjuggler_and_rviz](https://user-images.githubusercontent.com/1334122/139539771-f9e2b9d6-4d21-4145-8b82-1aabb3cc0b06.jpg)

regular rviz

---
![plots](https://user-images.githubusercontent.com/1334122/139539907-f18df1eb-70ce-4aa6-8c48-ffc0ab7d02f7.jpg)
Another plotjuggler + custom tool

---

![custom_gui](https://user-images.githubusercontent.com/1334122/139539824-34627276-97b6-4dd5-8dc9-d07d44d543e7.png)

This wasn't plotjuggler, and windows instead of ubuntu?

Not sure which image was from which team, TUM is saying they're going to open source some or all of their stuff later, it'll be interesting to see what visualization tools they were using.

**Comment by facontidavide** [comment]
I love it. Thanks for sharing
